# taken from https://vision.in.tum.de/data/datasets/rgbd-dataset/file_formats
# freiburg #1
orb_slam:
  camera:
    preprocess: False
    rgb_topic: "/camera/rgb/image_color"
    fps: 30
    width: 640
    height: 480
    fov_x: 518 # example
    fov_y: 519 # example
    focal_x: 517.3
    focal_y: 516.5
    center_x: 318.6
    center_y: 255.3
    dist_coeffs: [
      0.2624,
      -0.9531,
      -0.0054,
      0.0026,
      1.1633
    ]

  depth_camera:
    rgb_topic: "/camera/rgb/image_color"
    rgb_info_topic: "/camera/rgb/camera_info"
    depth_topic: "/camera/depth/image"
    depth_info_topic: "/camera/depth/camera_info"
    fps: 30
    width: 640
    height: 480
    fov_x: 518 # example
    fov_y: 519 # example
    focal_x: 517.3
    focal_y: 516.5
    center_x: 318.6
    center_y: 255.3
    dist_coeffs: [
      0.2624,
      -0.9531,
      -0.0054,
      0.0026,
      1.1633
    ]
    fps_depth: 30
    width_depth: 640
    height_depth: 480
    focal_x_depth: 591.1
    focal_y_depth: 590.1
    center_x_depth: 331.0
    center_y_depth: 234.0
    dist_coeffs_depth: [
      -0.0410,
      0.3286,
      0.0087,
      0.0051,
      -0.5643
    ]